walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,6 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
-    return "Hello " + name;
+    return "Hello " + name.trim().replaceAll("[^a-zA-Z0-9!?,.:]", "");
```

Comment:
If you changed the implementation to remove spaces and special characters - prefer to mention this in the method's JavaDoc.